_data/apis.yml

```diff
@@ -224,23 +339,56 @@
 
 - name: Lingo24
   id: lingo24
+  company: unbabel
   urls:
    - https://www.lingo24.com
    - https://developer.lingo24.com/documentation
   privacy_url: https://www.lingo24.com/about/privacy-and-terms/
   languages: [ ar, da, de, en, es, fr, it, nl, pl, pt-br, pt-pt, ru, sv, tr ]
   self-serve: true
+  active: false
+
+- name: Lingua Custodia
+  id: lingua-custodia
+  company: lingua-custodia
+  urls:
+    - https://www.linguacustodia.finance/en/welcome/
+  privacy_url:
+    - https://www.linguacustodia.finance/en/lingua-custodia-privacy-policy/
+  stock_api_domains:
+    - finance
+  self-serve: false
 
 - name: LingvaNex
   id: lingvanex
+  company: lingvanex
   urls:
    - https://lingvanex.com
    - https://docs.lingvanex.com/reference/user-guide
   languages: [ af-za, sq-al, am-et, ar-sa, hy-am, az-az, eu-es, be-by, bn-bd, bs-ba, bg-bg, ca-es, ceb-ph, ny-mw, zh-Hans-cn, zh-Hant-tw, co-fr, ht-ht, hr-hr, cs-cz, da-dk, nl-nl, en-us, eo, et-ee, fi-fi, fr-ca, fy-nl, gl-es, ka-ge, de-de, el-gr, gu-in, ha-ne, haw-us, he-il, hi-in, hmn-cn, hu-hu, is-is, ig-ng, id-id, ga-ie, it-it, ja-jp, jv-id, kn-in, kk-kz, km-kh, rw-rw, ko-kr, ku-ir, ky-kg, lo-la, la-vat, lv-lv, lt-lt, lb-lu, mk-mk, mg-mg, ms-my, ml-in, mt-mt, mi-nz, mr-in, mn-mn, my-mm, ne-np, 'no-no', or-or, ps-af, fa-ir, pl-pl, pt-pt, pa-pk, ro-ro, ru-ru, sm-ws, gd-gb, sr-Cyrl-rs, st-ls, sn-zw, sd-pk, si-lk, sk-sk, sl-si, so-so, es-es, su-id, sw-tz, sv-se, tl-ph, tg-tj, ta-in, tt-tt, te-in, th-th, tr-tr, tk-tk, uk-ua, ur-pk, ug-ug, uz-uz, vi-vn, cy-gb, xh-za, yi-il, yo-ng, zu-za ]
   self-serve: true
 
+- name: LT Gear
+  id: ltgear
+  company: moravia
```

Comment:
RWS, no?

Reply:
OK, sure.
Shall we bypass the `Moravia` entry in `company.yml` in this case? I still use it for `MoraviaMT`.